### ISSUE TITLE
Finalising ventilation and air conditioning

### ISSUE THREAD

**Issue by iteles** [dwyl/home#99]
Now that we're finally at the tail end of the work that needs to be done here, I wanted to put together a final list of things that need to be done for the ventilation and air conditioning (both construction and administrative) so that we can track/update them.

## Final tasks for ventilation and air conditioning
+ [x] Check contract for payments terms against values of final invoices
+ [x] Review final invoice against original quotes
+ [x] Speak to Francisco on any discrepancies/questions & follow up with email
+ [x] Receive confirmation on:
  + [x] Reasons for increase in price for external machines
  + [x] Final figures for air inlets #21 
  + [x] Detailed breakdown of first invoice
+ [x] Organise timing for cutting down the ventilation tubing to size (**high priority request from Pinheiro** who's afraid they're going to scuff up the paint on the ceilings)
+ [x] Agree timelines to finalise ventilation motor in attic
+ [x] Agree timelines for when the splits will be put against the walls
+ [x] Agree timelines for switching on and testing the whole system
+ [x] System switched on and fully tested
+ [ ] Understand when the ventilation in the bathrooms will get switched on , as in, is it when the lights switch on in the bathrooms? Is it something that happens every x minutes automatically?

**Comment by iteles** [comment]
I have gone through all quotes since the beginning and noted the differences from the final invoice.

They are as follows:
+ The external machines for each floor have gone up by about €180 per floor - I suspect that this is due to the fact that it's been over a year since we got the quote
+ The air inlets #21 are still priced at the original quote price which included opening holes in walls and more expensive inlets (before the solution in #21 was found)
+ This invoice takes us to 95% of the total value of the work when there are still a few small things to be finished and - most importantly - the whole system needs to be tested
  + I have told them that we'll pay up to 85% of the value at this stage and will only be paying the rest once everything is complete and all equipment has been tested
 

**Task:** Tomorrow when we're on-site we'll do a final check of all tasks to be completed by the climatization folks and send them the final list.

**Comment by iteles** [comment]
Final list of things that need to be finished by the AVAC folks, updated in the OP above:
+ Cutting the tubing and covering them with the appropriate exhausts
+ Finalising ventilation motor in attic
+ 

![IMG_4466](https://user-images.githubusercontent.com/4185328/68088388-a0354900-fe56-11e9-8b09-b048a448cf5c.JPG)

Then the final point is that we'd originally agreed to have 'grills' on the underside of the shutter boxes which currently have just the air inlets #21 on the top side (can't be seen in the photo below) and the holes for these:
![IMG_4465](https://user-images.githubusercontent.com/4185328/68088801-8dbd0e80-fe5a-11e9-942a-dcd6596ada39.JPG)

But in actual fact, I quite like them just as they are, without additional chrome to clean.

@nelsonic Any preference to have the grills _on_?

**Comment by nelsonic** [comment]
@iteles the question is not one of preference or aesthetics for me; 
it's really a question of sound and thermal insulation.
Can we _test_ both and _then_ decide?

**Comment by iteles** [comment]
I haven't received an answer to my email yet but I have spoken to Francisco a couple of times. They have started on dealing with the ventilation shafts but haven't so far finished.

I have been assured that this will be concluded in the coming week (beg. 11-Nov-2019).

In the machine room, the shaft looks like this:
![IMG_4622](https://user-images.githubusercontent.com/4185328/68499674-ad38aa80-0251-11ea-8742-f5386932bdac.JPG)

I am keen to understand _when_ the ventilation will get switched on , as in, is it when the lights switch on in the bathrooms? Is it something that happens every x minutes automatically?

**Comment by nelsonic** [comment]
@iteles we _definitely_ don't want the ventilation switching on/off intermittently ... so good to ask/check. 👍

**Comment by iteles** [comment]
So far, it would appear that the ventilation is on 100% of the time 🤦‍♀ 
A solution will need to be found for this ridiculousness.

**Comment by iteles** [comment]
Timelines now agreed for switching on and testing the system: this should be done today.

**Comment by iteles** [comment]
Glad we have this issue because I've just received a very nice email asking why we haven't paid the final instalment (€12,500 🤯 ) and these are all the reasons why, we were still waiting for answers from them.

On this now.

**Comment by nelsonic** [comment]
How much?! 😮

**Comment by iteles** [comment]
There was actually an outstanding amount of 3.013,94€ left to pay. This is actually **less than what is outstanding in our payment spreadsheet**: €4,200, so it's not a bad thing. (This is due to them not charging us IVA on the first invoice so it balances out to a lower amount than we expected in the end).

I am seeing if there's anything that can be done about auto-liquidação de IVA on this invoice and if not, I will charge it through ideaQ (I will check as it may have been charged through DWYL already).
Either way, it will get paid in the next few days.

# tl;dr


## The work that should have been cheaper
We exchanged the original proposal (carotes - holes in the walls) for the cheaper holes in the window boxes for ventilation, but never got the orçamentos adjusted for this (because I kept asking and nothing was forthcoming).

On speaking to Francisco he said that the work they ended up having to do because they were going to put the external machines on one end of the roof and ended up having to move them - both labour and extra tubing - outweighs the change in cost, which I do kind of believe.

So we have left this as in - one piece of work which should be cheaper exchanged for another that should be more expensive - and called it even.

## Increased price of splits

There was a difference here to our original orçamento because of a revised orçamento sent on 30th January 2018.

When I found and opened the email, I had a draft email fully written out as below and not sent:
<img width="1500" alt="image" src="https://github.com/dwyl/home/assets/4185328/14e608f2-a6a9-40b9-a55d-62c842cc04a1">

This was before I forgot to do things like this, when I was still on my game about admin stuff like this. So if it's not sent it's because we decided not to send it. Annoyingly, I can't find a record of this anywhere.

**Comment by iteles** [comment]
<img width="524" alt="image" src="https://github.com/dwyl/home/assets/4185328/2c6a4835-e22a-412c-b7ee-88db3e343de0">

**Comment by nelsonic** [comment]
How on earth did we still owe them this much? And why didn’t you get them to agree to maintaining the machines that vibrate too much *before* this transfer? 💭
